// @slidev/parser/dist/core.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});
















var _chunkGA7DXWKMjs = require('./chunk-GA7DXWKM.js');

















exports.detectFeatures = _chunkGA7DXWKMjs.detectFeatures; exports.filterDisabled = _chunkGA7DXWKMjs.filterDisabled; exports.getDefaultConfig = _chunkGA7DXWKMjs.getDefaultConfig; exports.mergeFeatureFlags = _chunkGA7DXWKMjs.mergeFeatureFlags; exports.parse = _chunkGA7DXWKMjs.parse; exports.parseAspectRatio = _chunkGA7DXWKMjs.parseAspectRatio; exports.parseRangeString = _chunkGA7DXWKMjs.parseRangeString; exports.parseSlide = _chunkGA7DXWKMjs.parseSlide; exports.prettify = _chunkGA7DXWKMjs.prettify; exports.prettifySlide = _chunkGA7DXWKMjs.prettifySlide; exports.resolveConfig = _chunkGA7DXWKMjs.resolveConfig; exports.resolveFonts = _chunkGA7DXWKMjs.resolveFonts; exports.scanMonacoModules = _chunkGA7DXWKMjs.scanMonacoModules; exports.stringify = _chunkGA7DXWKMjs.stringify; exports.stringifySlide = _chunkGA7DXWKMjs.stringifySlide; exports.verifyConfig = _chunkGA7DXWKMjs.verifyConfig;
