// @slidev/cli/dist/index.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});










var _chunkTH7JXF42js = require('./chunk-TH7JXF42.js');


var _chunkRSZHGGR3js = require('./chunk-RSZHGGR3.js');


var _chunk4O6GOAOCjs = require('./chunk-4O6GOAOC.js');
require('./chunk-KKGXM3XL.js');
require('./chunk-34SGGZ4P.js');
require('./chunk-QDKRARRU.js');













exports.ViteSlidevPlugin = _chunk4O6GOAOCjs.ViteSlidevPlugin; exports.createServer = _chunkTH7JXF42js.createServer; exports.createWindiCSSPlugin = _chunkRSZHGGR3js.createWindiCSSPlugin; exports.getAddonRoots = _chunkTH7JXF42js.getAddonRoots; exports.getCLIRoot = _chunkTH7JXF42js.getCLIRoot; exports.getClientRoot = _chunkTH7JXF42js.getClientRoot; exports.getRoot = _chunkTH7JXF42js.getRoot; exports.getThemeRoots = _chunkTH7JXF42js.getThemeRoots; exports.getUserRoot = _chunkTH7JXF42js.getUserRoot; exports.isPath = _chunkTH7JXF42js.isPath; exports.parser = _chunkTH7JXF42js.parser; exports.resolveOptions = _chunkTH7JXF42js.resolveOptions;
